src/pages/LandingPage.tsx

```diff
@@ -82,21 +82,21 @@ export const LandingPage = () => {
       <div className='header-subtext'>
         <h3>Discover Everything Tech in Egypt 🚀 — Your Ultimate Insight Hub!</h3>
       </div>
-      <Divider />
+      <hr className='divider' />
```

Comment:
I think you can encapsulate this line to a component  with name `Divider`

it's gonna be more readable, or even in the future if we want to have different variations of divider we can just edit one component.